VersionControlUI/src/nodes/DiffFrame.h

```diff
@@ -66,21 +66,30 @@ class VERSIONCONTROLUI_API DiffComparisonPair : public Super<Visualization::UINo
 		Model::Text* singleObjectPath();
 
 		bool twoObjectPathsDefined();
-		bool isDummyDiffComparisonPair();
+		bool isDummyDiffFrame();
 		QString comparisonName();
 
 		QList<ObjectPathCrumbData> objectPathCrumbsDataOldNode();
 		QList<ObjectPathCrumbData> objectPathCrumbsDataNewNode();
 
 		static QString computeObjectPath(Model::Node* node);
 
+		void addOldChangedNode(Model::Node* oldNode);
```

Comment:
Yes it was accidental, since the rename just shows the whole file as new. 

OK, I'll try to do this.